openapi.yaml

```diff
@@ -28522,6 +28953,30 @@ components:
             The date this key was added.
           example: "2018-01-01T00:01:01"
           readOnly: true
+    UserType:
+      type: object
+      description: >
+        The type of user on an account. Mostly applies to the use of the parent and child accounts for Akamai partners functionality.
+      properties:
+        user_type:
+          type: string
+          readOnly: true
+          enum:
+          - parent
+          - child
+          - proxy
+          - default
+          description: |
+            If the user belongs to a [parent or child account](/docs/guides/parent-child-accounts/) relationship, this defines the user type in the respective account. Possible values include:
+
+            * `parent`. This is a user in an Akamai partner account. Akamai partners have a contractural relationship with their end customers, to sell Akamai services. This user can either have full access (a parent account admin user) or limited access. Limited users don't currently have access to manage child accounts, but they can be granted this access by an admin user.
+
+            * `child`. This is an Akamai partner's end customer user, in a child account. A child user can have either full or limited access. Full access lets the user manage other child users and the proxy user in a child account. All child users can monitor the proxy user's activities in the child account.
+
+            * `proxy`. This is a user on a child account that gives parent account users access to that child account. A parent account admin user can [create a bearer token](#proxy-user-token-create). The parent user can use this token to run API operations from the child account, as if they were a child user.
```

Comment:
> A parent account admin

They need not be an admin, all that's required is the `child_account_access` grant. It might also be worth making this section a little more clear, in that a token can be created on the parent side _for_ this proxy user.